contentful/content-migrations/helpers/validate-environment.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function validateEnvironment() {
+	const allowedEnvironments = ['PlanTech_DevAndTest', 'PlanTech_StagingAndProduction'];
```

Comment:
Anyway to do this programatically? I'd imagine we _might_ change the names once we combine the C&S environments, so would be good to keep it _neat_

Reply:
I was going to suggest not but actually I think this is a very good idea - the reason this validation was added is because contentful defaults to master if you have a typo in the environment (which still makes me very very sad) so if we changed the names we could have a right mess with stuff happening on the master branch

I've setup the management scripts to do this programatically and removed it from the migration ones because those do a plan first and you can see if the env is wrong (/ I think the migration one validates it)